src/window.rs

```diff
@@ -607,6 +609,7 @@ where
                 } => {
                     behavior.touchpad_rotate(self, device_id, delta, phase);
                 }
+                WindowEvent::ActivationTokenDone { .. } => todo!(),
```

Comment:
Not sure if and how we should handle this event

Reply:
Appit is a thin wrapper for Winit. It should expose everything as closely to winit as possible.